src/plugins/intel_cpu/src/nodes/rnn.cpp

```diff
@@ -255,25 +256,57 @@ bool RNN::isSupportedOperation(const std::shared_ptr<const ov::Node>& op, std::s
         }
 
         auto rnnCellBase = ov::as_type_ptr<const ov::op::util::RNNCellBase>(op);
-        if (rnnCellBase && rnnCellBase->get_clip() != 0.f) {
-            errorMessage = "Clipping is not supported for RNN primitive.";
-            return false;
+        if (rnnCellBase) {
```

Comment:
I believe we can add an early return if cast to RNNCellBase is failed.

Reply:
It would be an issue for v0::LSTMSequence since it doesn't inherit from RNNCellBase.